docs/site/tutorials/a_swift_tour.ipynb

```diff
@@ -842,21 +999,23 @@
           "metadata": {
             "tags": []
           },
-          "execution_count": 29
+          "execution_count": 27
         }
       ]
     },
     {
       "metadata": {
-        "id": "aVAMxeTxdE2v",
-        "colab_type": "text"
+        "id": "sgYex6gigH87",
+        "colab_type": "code",
+        "colab": {}
       },
-      "cell_type": "markdown",
+      "cell_type": "code",
```

Comment:
Since this cell is just comments, does it make sense for this to be converted back to markdown? Or is there another reason for it to be a code cell?

(Note: this pattern of changes is repeated in other cells too.)

Reply:
"A Swift Tour" has experiment sections, which are exercises for the reader.

Example:
![image](https://user-images.githubusercontent.com/5590046/54781138-b698d800-4bd8-11e9-87ea-ef324816074c.png)

The original "A Swift Tour" is a static page, so everything shows up as text.
Since we're porting it to an interactive notebook, I think it makes sense to translate these into code sections. Users can actually code and experiment!

![image](https://user-images.githubusercontent.com/5590046/54781289-15f6e800-4bd9-11e9-8bd7-504f2b659a38.png)